lib/paths.cf

```diff
@@ -25,6 +25,22 @@ bundle common paths
 #        comment => "Alternate array reference style";
 # }
 # ```
+#
+# Additionally a `root_path` entry is present in the `path` array to aid in policy
```

Comment:
in the `path` array, or in the `paths` bundle?

Reply:
oh, right, bundle.